lib/kennel/unmuted_alerts.rb

```diff
@@ -11,12 +11,16 @@ class UnmutedAlerts
     }.freeze
 
     class << self
-      def print(api, tag)
+      def print(api, tag, scope)
         monitors = filtered_monitors(api, tag)
         if monitors.empty?
           Kennel.out.puts "No unmuted alerts found"
         else
           monitors.each do |m|
+            if scope
+              m[:state][:groups].select! { |g| g[:name].include?(scope) }
```

Comment:
`"foo,bar".include?("ba")` would be true, so we should split before testing
```suggestion
              m[:state][:groups].select! { |g| g[:name].split(",").include?(scope) }
```